practice/timezone/timezone/logic.py

```diff
@@ -0,0 +1,41 @@
+"""timezone Logic."""
+
+from tzwhere import tzwhere
+import arrow
+
+
+def get_time_at_tz(tz_string):
+    """Return the current time (as a string) at a given timezone."""
+    utc = arrow.utcnow()
+    time_string = str(utc.to(tz_string))
+    return time_string
+
+
+def get_tz(lat_lng_list):
+    r"""Return the time zone given lat, long.
+
+    >>> get_tz([45,45])
+    'Europe/Moscow'
+    """
+    tz = tzwhere.tzwhere()
+    tz_string = tz.tzNameAt(float(lat_lng_list[0]), float(lat_lng_list[1]))
```

Comment:
Similarly, this function also deals with serialization. I would argue that this function should throw an error if a list of strings is given to it. The most native type that logic should deal with is a list of floats. It's the job of the view to prepare any input data (say the `lat,lng` string in the URL) fully before it gets here.